Issue with @types/tapable breaking webpack in playground

Looks like a new release of `@types/tapable` have broken the build
```[at-loader] Checking finished with 1 errors
Error in bail mode: [at-loader] ../../node_modules/@types/webpack/index.d.ts:469:38 
    TS2694: Namespace '"/home/travis/build/hyperledger/composer/node_modules/@types/tapable/index"' has no exported member 'Plugin'.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! composer-playground@0.18.1 build:prod: `webpack --config config/webpack.prod.js  --progress --profile --bail`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the composer-playground@0.18.1 build:prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-03-12T08_31_27_979Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! composer-playground@0.18.1 e2e:main: `npm run build:prod && node ./e2e/test.js && npm run e2e:fabric`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the composer-playground@0.18.1 e2e:main script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-03-12T08_31_27_993Z-debug.log```

Solution is to add the `"@types/tapable": "0.2.5",`to the package.json of playground

--- a/packages/composer-playground/package.json
+++ b/packages/composer-playground/package.json
@@ -134,6 +134,7 @@
         "@types/source-map": "0.5.0",
         "@types/uglify-js": "2.6.28",
         "@types/webpack": "2.1.0",
+        "@types/tapable": "0.2.4",
         "angular-2-local-storage": "1.0.0",
         "angular-router-loader": "0.5.0",
         "angular2-template-loader": "0.6.2",

